### DISCUSSION TITLE
Compilation

### DISCUSSION THREAD

**Discussion by sneekyfoxx** [modular/modular#148] [Q&A] [answered]
Can Mojo be compiled to an executable? if so, would it be safe to assume that a Mojo file can be cross-compiled to any architecture?

**Comment by lattner** [comment, marked as answer]
Yes, it generally supports compiling to anything that has an LLVM or MLIR backend.

**Reply by sneekyfoxx** [reply]
Thank you! I'm looking forward to a production ready Mojo. Python is such a simple, yet powerful language. A faster and compiled superset of it brings excitement.
